src/cmds/eval/evalWorker.ts

```diff
@@ -22,6 +22,13 @@ if (parentPort !== null) {
     compartment.evaluate(
       // '() => ' + readFileSync(pluginFilePath, 'utf8')
       readFileSync(pluginFilePath, 'utf8'),
+      {
+        sloppyGlobalsMode: true,
+        __moduleShimLexicals__: undefined,
+        __evadeHtmlCommentTest__: false,
+        __evadeImportExpressionTest__: true,
+        __rejectSomeDirectEvalExpressions__: false,
+      },
```

Comment:
We aren't going to use these options in prod, so this entire config is going to have to go, and (some of) the transforms re-enabled.

However, we should PR the changes to the transforms separately, so for now, we can do whatever works. I'd prefer if we deleted these lines and uncommented the transforms in `postProcess`, if possible.